detection_rules/packaging.py

```diff
@@ -202,15 +207,25 @@ def _package_index_file(self, save_dir):
     def save_release_files(self, directory, changed_rules, new_rules, removed_rules):
         """Release a package."""
         summary, changelog = self.generate_summary_and_changelog(changed_rules, new_rules, removed_rules)
-
         with open(os.path.join(directory, f'{self.name}-summary.txt'), 'w') as f:
             f.write(summary)
         with open(os.path.join(directory, f'{self.name}-changelog-entry.md'), 'w') as f:
             f.write(changelog)
-        with open(os.path.join(directory, f'{self.name}-consolidated.json'), 'w') as f:
-            json.dump(json.loads(self.get_consolidated()), f, sort_keys=True, indent=2)
+
+        consolidated = json.loads(self.get_consolidated())
+        with open(os.path.join(directory, f'{self.name}-consolidated-rules.json'), 'w') as f:
+            json.dump(consolidated, f, sort_keys=True, indent=2)
+        with open(os.path.join(directory, f'{self.name}-consolidated-rules.ndjson'), 'w') as f:
+            f.write('\n'.join(json.dumps(r, sort_keys=True) for r in consolidated))
```

Comment:
we need a `utils.ndjson(list)` function by now. to remove all of the `'\n'.join(...)` pattern usage

Reply:
added in e9445842eaa980dbdfe823760dc990e748c99fa9

Will ensure that all paths are using `Path` in #360 and then do a subsequent PR to use `Ndjson` everywhere else in the repo